src/components/pages/Offer/Offer/Stocks/StockItem/domain.js

```diff
@@ -0,0 +1,30 @@
+export const validateCreatedStock = stock => {
+  let errors = []
+
+  if (stock.price < 0) {
+    errors['price'] = 'Le prix doit être positif.'
```

Comment:
Si on part sur un object plutôt qu'un array, on pourrait alléger l'écriture 
```suggestion
    errors.price = 'Le prix doit être positif.'
```
Cela vaut pour les suivants bien sur